### ISSUE TITLE
automate creating the cluster account

### ISSUE THREAD

**Issue by cdkharris** [UCL-ARC/terraform-aws-vslurm#12]
Creating the cluster with `sacctmgr` is still manual:

`sacctmgr create cluster vslurm`, then input 'y'

so i would like to
- check the man page for a way to automate it
- find a way to make ansible do it idempotently

**Comment by cdkharris** [comment]
this is the current manual procedure for running vslurm:

1. i run `make ec2-vslurm`
2. i copy the ansible dir onto the deployer with `scp -ri ./ec2_id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ./../ansible <deployer_username_and_host>:/home/ec2-user`
3. i log in to the deployer (bootstrapper?) as ec2-user
4. i do `sudo mv ansible /root/`
5. i do `sudo su -`
6. i do` cd ./ansible`
7. i run the ansible playbook.
  - `ansible-playbook cluster.yaml --tags common_init`
  - `ansible-playbook cluster.yaml --tags slurm_common`
  - `ansible-playbook cluster.yaml --tags slurm_server`
8. i do `ssh ec2-user@server "sudo sacctmgr create cluster vslurm"`
9. i do `ansible-playbook cluster.yaml --tags slurm_compute`
10. i do `ansible-playbook cluster.yaml --tags slurm_login`
11. i do `ansible all -m ansible.builtin.reboot`
12. and then `ansible-playbook cluster.yaml`

Steps 2-7 are automated in `ec2-vslurm/scripts/deployer_user_data`.
Basically, I'm happy running the ansible commands there.

There are two tasks that are still truly manual as of 70d5e5c, both to do with the accounting database:

- step 8 described above, which asks for 'y' input
- and getting the encoded password to put into the mysql user creds in `ansible/slurm_server/tasks/main.yaml`

**Comment by cdkharris** [comment]
the input is not a problem; `sacctmgr` takes the `-i` option. i set up a task that bases its failure/changed status on the return code for `sacctmgr -i create cluster`. Fixed with f011d14e7ea32ec207f225cb780c56d7c8cd839f
